packages/tools/devtools/devtools-core/src/test/DefaultVisualizers.spec.ts

```diff
@@ -1730,6 +1730,380 @@ describe("DefaultVisualizers unit tests", () => {
 		expect(result).to.deep.equal(expected);
 	});
 
+	it.only("SharedTree: Renders multiple allowed types in SharedTree's root field", async () => {
```

Comment:
Will be removing `only()` (and the one below), once we agree on the structure of the visualization (especially adding the `root` layer).